MIL/C10_Topology/S03_Topological_Spaces.lean

```diff
@@ -214,7 +214,7 @@ BOTH: -/
 Then the next big piece is a complete lattice structure on ``TopologicalSpace X``
 for any given structure. If you think of topologies as being primarily the data of open sets then you expect
 the order relation on ``TopologicalSpace X`` to come from ``Set (Set X)``, i.e. you expect ``t ≤ t'``
-if a set ``u`` is open for ``t'`` as soon as it is open for ``t``. However we already know that Mathlib focuses
+if a set ``u`` is open in ``t'`` only if it is open for ``t``. However we already know that Mathlib focuses
```

Comment:
This is the one sentence where I think my change should be double checked. The previous wording did not really make sense to me.

Reply:
This change is not correct, please revert it. If you find it clearer, you can write “with respect to `t`” instead of “for `t`”. But “open in `t`” is really weird. And the `only if` is definitely backward.

Reply:
Reverted.